tests/mcp/test_resources.py

```diff
@@ -30,6 +36,32 @@ def test_rules_sorted_by_id(self):
         rule_ids = [r["rule_id"] for r in rules]
         assert rule_ids == sorted(rule_ids)
 
+    def test_rules_catalog_includes_deprecated(self):
+        """Test that rules catalog includes deprecated field."""
+        rules = _get_rules_catalog()
+        for rule in rules:
+            assert "deprecated" in rule
+            assert isinstance(rule["deprecated"], bool)
+
+    def test_rules_catalog_includes_added_in_version(self):
+        """Test that rules catalog includes added_in_version field."""
+        rules = _get_rules_catalog()
+        for rule in rules:
+            assert "added_in_version" in rule
```

Comment:
It looks like we have severla tesrs for single attribute. Wouldnt it make sense to combine those tests into one? And test it once (ie if deprecated or added_in_version are in return)